docs/src/main/sphinx/functions/session.rst

```diff
@@ -8,4 +8,12 @@ Functions providing information about the query execution environment.
 
     Returns the current user running the query.
 
+.. data:: current_catalog
+
+    Returns a character string that represents the current default catalog name.
```

Comment:
remove the word "default"

Reply:
("current default" would mean that which is currently set as a default somewhere, which could be different that the current catalog. If current_catalog means what is set somewhere as a default, but you are looking at another catalog presently, then it is mis-named and should be called "default_catalog".

Reply:
Thanks for reviewing the PR and suggesting new changes. I have removed the word "default".